definitions/v4/drunkenslug.yml

```diff
@@ -180,8 +166,8 @@ search:
       attribute: value
 download:
   error:
-    - selector: error
-      message:
-        selector: error
-        attribute: description
+  - selector: error
+    message:
+      selector: error
+      attribute: description
```

Comment:
@bakerboy448  this is valid YML as is the previous - as long as Cardigann doesn't complain (@Qstick are you able to confirm?) then we should be fine